### ISSUE TITLE
Text does not wrap to second line but does on thrid

### ISSUE THREAD

**Issue by dka09** [zeit/hyperterm#64]
Text does not wrap when you reach the end of the window but when you expect text to be on the third line it does wrap and goes to the second line.

**Comment by rauchg** [comment]
Can you elaborate a bit more please?

**Comment by dka09** [comment]
Sorry I didn't explain to well. 

![line wrap](http://i.giphy.com/3oEjHM9HTtnBTmp4qI.gif)

**Comment by CodeTheory** [comment]
I believe this was fixed with the introduction of the `LANG` env variable as noted in #283.

**Comment by atdrago** [comment]
I believe this is still reproducible today, and its cause is having any colored text.

In Hyperterm,

1. `export PS1="\e[0;32m\$\e[m "`
2. Type until the end of the line and *observe it will not wrap*
3. `export PS1="\$ "`
4. Type until the end of the line and *observe text wraps correctly*

**Comment by trevordmiller** [comment]
I have a custom PS1 with encoded colors and am experiencing this as well still on the latest build

**Comment by liwsakilive** [comment]
@trevordmiller 
URL not working

**Comment by trevordmiller** [comment]
@liwsakilive Updated URL: https://github.com/trevordmiller/settings/blob/master/dotfiles/.bash_profile#L20-L29

I can no longer reproduce on latest Hyper; this must have been fixed with one of the releases in the past 8 months.

**Comment by unleashit** [comment]
Spoke to soon, the problem came back. I'll also jump on the screencast bandwagon since my issue is a bit of a variation from the the above. It's a bit different whether I'm in  zsh or bash, and notice what happens when I backspace:

![hyper](https://user-images.githubusercontent.com/5051359/27257830-21601762-539b-11e7-8e62-9cfe359d565e.gif)

Everything was fine until I added a couple of plugins to my .hyper.js without installing them first (I'm new to Hyper and didn't know that was a key step (duh), After seeing the notification that hyper wasn't happy, I installed the plugins. Didn't fix the wrapping issue, neither did reverting my .hyper.js to the original. I'm not saying adding the plugins is what triggered it, but that's exactly all I did before the problem began.

Unfortunately, this is all the time I can invest in Hyper. Hope you guys can get it sorted.
